code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -271,21 +271,31 @@
 	update_icon()
 
 	ai_log("Life() - stance=[stance] ai_inactive=[ai_inactive]", 4)
-	//Movement
-	if(!ai_inactive && !stop_automated_movement && wander && !anchored) //Allowed to move?
-		handle_wander_movement()
 
-	//Speaking
-	if(!ai_inactive && speak_chance && stance == STANCE_IDLE) // Allowed to chatter?
-		handle_idle_speaking()
-
-	//Stanceyness
+	//AI Actions
 	if(!ai_inactive)
+		//Stanceyness
 		handle_stance()
 
-	//Resisting out of things
-	if(!ai_inactive && incapacitated(INCAPACITATION_DEFAULT) && stance != STANCE_IDLE)
-		resist()
+		//Movement
+		if(!stop_automated_movement && wander && !anchored) //Allowed to move?
+			handle_wander_movement()
+
+		//Speaking
+		if(speak_chance && stance == STANCE_IDLE) // Allowed to chatter?
+			handle_idle_speaking()
+
+		//Resisting out buckles
+		if(incapacitated(INCAPACITATION_BUCKLED_PARTIALLY))

```

Comment:
This used to happen only if STANCE_IDLE.  Now its any stance? I suppose that makes sense, just checking.